dali/operators/image/color/brightness_contrast.cc

```diff
@@ -31,12 +32,12 @@ This operator can also change the type of data.)code")
     .NumOutput(1)
     .AddOptionalArg("brightness",
                     "Brightness mutliplier.",
-                    kDefaultBrightness, true)
+                    kDefaultBrightness, true, true)
```

Comment:
General remark, how about adding a dedicated enum for the PerFrame option rather than having the growing `true, true` chain. Ofc in another PR.

Reply:
Actually, I was thinking about `flags`.